README.md

```diff
@@ -24,13 +24,12 @@ Install Go from https://go.dev/dl/.
 
 ### Initial Project Setup            
 Run `git clone https://github.com/Charlene999/CEN3031-The-Perfect-Path.git` from a terminal or command line pathed into a designated folder.            
-Change directory into the newly created CEN3031-The-Perfect-Path folder.            
-Run `ng new application`.            
-&emsp; 	When it asks, add Angular routing and use CSS.            
+Change directory into the newly created CEN3031-The-Perfect-Path folder.             
 
 ### Frontend Setup            
 Change directory into application.            
```

Comment:
should we combine this step with the step above so that you just cd into CEN3031..../application?